website/docs/streaming-platforms-eks/nifi.md

```diff
@@ -0,0 +1,8 @@
+---
+sidebar_position: 3
+sidebar_label: Apache NiFi on EKS
+---
+
+# Apache NiFi on EKS
+
+### Coming soon..
```

Comment:
could you pls add instruction about how to deploy this example here?
I tried terraform init and apply but see ACM cert is required. Don't know how to proceed.
![image](https://user-images.githubusercontent.com/17306003/217304065-6733c67e-3d94-45bf-adf2-8c65fcc78c95.png)